build.gradle

```diff
@@ -30,21 +31,50 @@ repositories {
     mavenCentral()
     jcenter()
 
+    maven { url 'https://jitpack.io' }
     maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
     maven { url 'https://papermc.io/repo/repository/maven-public/' }
+    maven { url = "https://repo.extendedclip.com/content/repositories/placeholderapi/" }
+    maven { url = 'https://repo.extendedclip.com/content/repositories/placeholderapi/' }
+    maven {
+        credentials {
+            username "$ghUser"
+            password "$ghPass"
+        }
+        url = "https://maven.pkg.github.com/DumbDogDiner/buildtoolspackages/" }
+
+    maven {
+        credentials {
+            username "$ghUser"
+            password "$ghPass"
+        }
+        url = "https://maven.pkg.github.com/DumbDogDiner/closedsource-package-mirror/" }
 }
 
 dependencies {
     compileOnly 'org.projectlombok:lombok:1.18.16'
     annotationProcessor 'org.projectlombok:lombok:1.18.16'
 
-    compileOnly 'com.destroystokyo.paper:paper-api:1.16.4-R0.1-SNAPSHOT'
-    compileOnly 'net.md-5:bungeecord-api:1.16-R0.4-SNAPSHOT'
+    compileOnly 'com.dumbdogdiner.buildtools:paper:1.16.4-318-SNAPSHOT'
+    compileOnly 'io.github.waterfallmc:waterfall-api:1.16-R0.4-SNAPSHOT'
+
     implementation 'org.jetbrains:annotations:20.1.0'
     implementation 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.atlassian.commonmark:commonmark:0.15.2'
     implementation 'io.github.classgraph:classgraph:4.8.92'
     implementation 'com.github.seancfoley:ipaddress:5.3.3'
-    
+    implementation 'org.apache.httpcomponents:httpclient:4.5'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.0'
+
+    //Softdepend Plugins (not all are used yet)
+    compileOnly 'net.luckperms:api:5.2'
+    compileOnly 'com.github.MilkBowl:VaultAPI:1.7'
+    compileOnly 'me.clip:placeholderapi:2.10.6'
+    compileOnly 'com.dumbdogdiner.closedsource-package-mirror:stafffacilities:4.8.5'
+    compileOnly 'com.dumbdogdiner.closedsource-package-mirror:plotsquared-bukkit-premium:5.13.8'
+    compileOnly 'com.dumbdogdiner.closedsource-package-mirror:spartan:377.1'
+    compileOnly 'com.dumbdogdiner.closedsource-package-mirror:warpsystem:4.2.12'
```

Comment:
Ideally we'd make a module system for these, since putting them in the core API seems like a bad idea, so let's discuss something like this at some point